tests/unit/ui/components/TestShowMoreCard.js

```diff
@@ -57,4 +58,19 @@ describe('<ShowMoreCard />', () => {
     const rootNode = findDOMNode(root);
     expect(rootNode.textContent).toContain('Hello I am description');
   });
+
+  it('executes truncateToMaxHeight when it recieves props', () => {
+    const node = document.createElement('div');
+    let ref;
+    ReactDOM.render(<ShowMoreCardBase i18n={getFakeI18nInst()} ref={(e) => { ref = e; }} />, node,
+      () => {
+        jest.spyOn(ref, 'truncateToMaxHeight');
```

Comment:
We use sinon for spies since the API is a little bit more complete and intuitive.